data.json

```diff
@@ -511,8 +511,21 @@
         "name": "WebStorm",
         "icon_path": "icon_WebStorm.png",
         "emoji_name": "webstorm",
-        "channel_name": "webstorm",
-        "category_name": "Products",
+        "channels": [
+            {
+                "name": "webstorm-announcements",
+                "description": "Product announcements and updates"
```

Comment:
I think we might need a way to mark that this channel shouldn't have regular permissions?

There is a `read_only` flag that already exists that might be applicable here if reworked slightly?